optimum/habana/transformers/trainer.py

```diff
@@ -129,7 +139,7 @@
 OPTIMIZER_NAME_BIN = "optimizer.bin"
 SCHEDULER_NAME = "scheduler.pt"
 SCALER_NAME = "scaler.pt"
-
+FSDP_MODEL_NAME = "pytorch_model_fsdp"
```

Comment:
It seems FSDP_MODEL_NAME was introduced after Transformers v4.34. Let's stick to v4.34 for now as v4.35/4.36 is not supported, and I'll update everything when upgrading to newer Transformers version.

Reply:
ok, will do